[2024-10-02T17:25:55.829+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T17:25:55.848+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [queued]>
[2024-10-02T17:25:55.855+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [queued]>
[2024-10-02T17:25:55.856+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T17:25:55.868+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-11-10 00:00:00+00:00
[2024-10-02T17:25:55.874+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1461) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T17:25:55.875+0000] {standard_task_runner.py:63} INFO - Started process 1466 to run task
[2024-10-02T17:25:55.874+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-11-10T00:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpxxzsz7q4']
[2024-10-02T17:25:55.876+0000] {standard_task_runner.py:91} INFO - Job 100: Subtask transformar
[2024-10-02T17:25:56.073+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T17:25:56.142+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-11-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-10T00:00:00+00:00'
[2024-10-02T17:25:56.143+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T17:25:56.244+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T17:25:56.244+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/etl_pipeline.py", line 17, in transformar
    df_agrupada = df.groupby('ContinentExp')[['cases', 'deaths']].sum().reset_index()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/frame.py", line 8869, in groupby
    return DataFrameGroupBy(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/groupby/groupby.py", line 1278, in __init__
    grouper, exclusions, obj = get_grouper(
                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/groupby/grouper.py", line 1009, in get_grouper
    raise KeyError(gpr)
KeyError: 'ContinentExp'
[2024-10-02T17:25:56.255+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-11-10T00:00:00+00:00, execution_date=20231110T000000, start_date=20241002T172555, end_date=20241002T172556
[2024-10-02T17:25:56.266+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 100 for task transformar ('ContinentExp'; 1466)
[2024-10-02T17:25:56.289+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-02T17:25:56.439+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T17:25:56.442+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T18:57:52.590+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:57:52.622+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [queued]>
[2024-10-02T18:57:52.810+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [queued]>
[2024-10-02T18:57:52.811+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:57:52.826+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-11-10 00:00:00+00:00
[2024-10-02T18:57:52.832+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5615) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:57:52.833+0000] {standard_task_runner.py:63} INFO - Started process 5629 to run task
[2024-10-02T18:57:52.833+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-11-10T00:00:00+00:00', '--job-id', '1091', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpalablf9h']
[2024-10-02T18:57:52.835+0000] {standard_task_runner.py:91} INFO - Job 1091: Subtask transformar
[2024-10-02T18:57:52.878+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:57:52.972+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-11-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-10T00:00:00+00:00'
[2024-10-02T18:57:52.976+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:57:53.065+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:57:53.065+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:57:53.073+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-11-10T00:00:00+00:00, execution_date=20231110T000000, start_date=20241002T185752, end_date=20241002T185753
[2024-10-02T18:57:53.127+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:57:53.149+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:57:53.151+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:24:09.870+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:24:09.905+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [queued]>
[2024-10-02T19:24:10.090+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [queued]>
[2024-10-02T19:24:10.091+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:24:10.104+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-11-10 00:00:00+00:00
[2024-10-02T19:24:10.111+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=8989) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:24:10.112+0000] {standard_task_runner.py:63} INFO - Started process 8998 to run task
[2024-10-02T19:24:10.112+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-11-10T00:00:00+00:00', '--job-id', '2100', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp_jvd1_mj']
[2024-10-02T19:24:10.114+0000] {standard_task_runner.py:91} INFO - Job 2100: Subtask transformar
[2024-10-02T19:24:10.165+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-11-10T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:24:10.274+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-11-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-10T00:00:00+00:00'
[2024-10-02T19:24:10.277+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:24:10.460+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:24:10.461+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:24:10.475+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-11-10T00:00:00+00:00, execution_date=20231110T000000, start_date=20241002T192409, end_date=20241002T192410
[2024-10-02T19:24:10.526+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:24:10.559+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:24:10.562+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
